### ISSUE TITLE
install on debian buster

### ISSUE THREAD

**Issue by kanliot** [bibletime/bibletime#172]
some feedback for you "install on debian buster "  (sparkylinux) 
Don't think I'll be using this software because of missing buttons and lack of keyboard navigation.

https://pasteboard.co/HYuu0vI.png

install needed to run cmake as root,  after installing 
cmake
build-essential
qtbase5-dev
 qttools5-dev-tools
qttools5-dev-tools libqt5svg5-dev qtmultimedia5-dev
 qttools5-dev
libclucene-dev
python3-dev python3-pip python3-tk python3-lxml python3-six
python-dev python-pip python-tk python-lxml python-six
libsword-dev
libqt5webkit5-dev

also I had to cancel the book downloads since it was slow, and bibletime  didn't like that.

**Comment by Alan-R** [comment]
@kanliot Keyboard navigation works fine for me. For example, I define page-down and page-up to go to next/previous chapters. It comes with some keyboard shortcuts, but they weren't wired to my brain like page-up and page-down - so I defined my own.

**Comment by gholmlund** [comment]
I just installed BibleTime 2.11.2 from the repositories on Debian 10. I also compiled BibleTime master on Debian 10. They look and operate as I would expect them to work. I am closing this report.
